src/components/Block.jsx

```diff
@@ -9,6 +9,35 @@ export default React.createClass({
     block : React.PropTypes.object.isRequired
   },
 
+  getActions() {
+    let { app, block, first, last } = this.props
+
+    let actions = [
+      {
+        id       : 'moveUp',
+        label    : 'Move Up',
+        onClick  : app.prepare(move, block, -1),
+        disabled : first
+      },
+      {
+        id       : 'moveDown',
+        label    : 'Move Down',
+        onClick  : app.prepare(move, block, 1),
+        disabled : last
+      },
+      {
+        id      : 'destroy',
+        label   : 'Remove',
+        onClick : app.prepare(destroy, block.id)
+      }
+    ]
```

Comment:
Beyond that, I don't like that this object is in here. I should add a layer between `onClick` and the provided handler that passes in the editor instance and block.